job-server-extras/src/main/scala/spark/jobserver/python/PythonContextFactory.scala

```diff
@@ -134,10 +134,18 @@ trait DefaultContextLikeImplementations {
   override lazy val pythonPath: Seq[String] = pythonPaths(config)
 
   protected def pythonPaths(config: Config): Seq[String] = {
-    val envPaths = sys.env.get("PYTHONPATH").map(_.split(":").toSeq).getOrElse(Seq())
+    // If Spark driver not in cluster, we use SJSPYTHONPATH
+    val pythonPath = if (!sparkContext.deployMode.equals("cluster")) {
+      sys.env.get("SJSPYTHONPATH")
```

Comment:
I am not using Python part, but I don't see any mention of `SJSPYTHONPATH` in documentation or anywhere else before.
Wouldn't it be breaking/unexpected change of the python path?

Reply:
Ah, sorry. I see you define it:
```
      // If the Spark driver is not in cluster, use the PYTHONPATH of client process
      val sjsPythonPath = environment.get("PYTHONPATH", "")
      env.put("SJSPYTHONPATH", sjsPythonPath)
```

Don't pay attention to my comment :)